### PR TITLE
ptolmic wonder

### PR DESCRIPTION
**Pull request by TheShadowOfHassen**

### CODE REVIEW

File: 0-ad-history-mod-package/simulation/templates/structures/ptol/wonder.xml [hunk 1]
```diff
@@ -7,6 +7,10 @@
   <Identity>
     <Civ>ptol</Civ>
     <SpecificName>Hieron tou Hōrou</SpecificName>
+    <History>
+        Temple of Edfu was built during the Ptolemaic reign of Egypt. However, the structure was built in traditional Egyptian style and dedicated to Horus, a traditional Egyptian deity.
+        Construction of the temple began in 237 BC, over the site of a previous temple to Horus. The temple was finished  in 57 BC and had the area of 64,626 square feet (6004 square meters)
```
Comment (TheShadowOfHassen) [inline 0-ad-history-mod-package/simulation/templates/structures/ptol/wonder.xml:12]:
```suggestion
        Construction of the temple began in 237 BC, over the site of a previous temple to Horus. The temple was finished  in 57 BC and had an area of 64,626 square feet (6004 square meters)
```

File: 0-ad-history-mod-package/simulation/templates/structures/ptol/wonder.xml [hunk 2]
```diff
@@ -7,6 +7,10 @@
   <Identity>
     <Civ>ptol</Civ>
     <SpecificName>Hieron tou Hōrou</SpecificName>
+    <History>
+        The Temple of Edfu was built during the Ptolemaic reign of Egypt. However, the structure was built in traditional Egyptian style and dedicated to Horus, a traditional Egyptian deity.
+        Construction of the temple began in 237 BC, over the site of a previous temple to Horus. The temple was finished  in 57 BC and had an area of 64,626 square feet (6004 square meters)
```
Comment (indoptogopt) [inline 0-ad-history-mod-package/simulation/templates/structures/ptol/wonder.xml:12]:
```suggestion
        Construction of the temple began in 237 BC, over the site of a previous temple to Horus. The temple was finished  in 57 BC and had an area of 64,626 square feet (6004 square meters).
```

**Review by indoptogopt — CHANGES_REQUESTED** [review]

**Comment by TheShadowOfHassen** [comment]
Is this Ok to merge?

**Comment by indoptogopt** [comment]
Yes.

**Pull request merged**